### PR TITLE
fix: repalce imgur with s3 for russian guide without conflict

### PR DESCRIPTION
**Pull request by ahmaxed**
(cherry picked from commit ac90750e36b0dd1fe508c69a2277b75be48e4b95)

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Reference #36007

### CODE REVIEW

File: guide/russian/cplusplus/tokens-variables/index.md [hunk 1]
```diff
@@ -21,8 +22,8 @@ localeTitle: Токены Часть 1
 ### Что такое переменные?
 
 *   Определение учебника: переменные называются ячейками памяти, данные которых могут быть изменены.
-    
-*   Но я хотел бы, чтобы вы подумали, что переменная будет чем-то вроде коробки, что-то вроде этого: ![Img](https://i.imgur.com/YdbgWHL.png)
+    https://cdn-media-1.freecodecamp.org/imgr/
```
Comment (RandellDawson) [inline guide/russian/cplusplus/tokens-variables/index.md]: This seems like it does not belong here.
```suggestion
```

File: guide/russian/cplusplus/tokens-variables/index.md [hunk 2]
```diff
@@ -176,8 +177,8 @@ int 9a;
 Если переменная не инициализирована, она содержит значение мусора. Например:
 
 Итак, с точки зрения ящиков, вы можете представить это как -
-
-![Img](https://i.imgur.com/YdbgWHL.png)
+https://cdn-media-1.freecodecamp.org/imgr/
```
Comment (RandellDawson) [inline guide/russian/cplusplus/tokens-variables/index.md]: This seems like it does not belong here.
```suggestion
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@ahmadabdolsaheb See my requested changes.

**Comment by ahmadabdolsaheb** [comment]
thanks @RandellDawson, your suggestions have been implemented.

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
@ahmadabdolsaheb OK, this PR looks good now.

**Comment by ahmadabdolsaheb** [comment]
I double checked this. it should be good to go.

**Comment by ahmadabdolsaheb** [comment]
@thecodingaviator, how does this look?

File: guide/russian/css/properties/background-position-property/index.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ localeTitle: Исходная позиция
 ```css
 /* setting background-image of HTML doc */ 
  body { 
-  background-image: url('https://i.imgur.com/6Z2VStD.png'); 
+  background-image: url('https://cdn-media-1.freecodecamp.org/imgr/6Z2VStD.png');
```
Comment (thecodingaviator) [inline guide/russian/css/properties/background-position-property/index.md]: Just putting it out here that we need to get on this when we get to the curriculum PR

**Review by thecodingaviator — APPROVED** [review]
Rest LGTM

**Comment by thecodingaviator** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**